src/declarations/stencil-public-runtime.ts

```diff
@@ -1031,6 +1031,7 @@ export namespace JSXBase {
     minlength?: number | string;
     multiple?: boolean;
     name?: string;
+    onSelect?: (event: Event) => void;
```

Comment:
Can you add the property for `onselect` here as well? (where 'onselect' is all lowercase)
```suggestion
    onSelect?: (event: Event) => void;
    onselect?: (event: Event) => void;
```